### ISSUE TITLE
feature concatenation about the paper, and the output layer in the actor network

### ISSUE THREAD

**Issue by Elfits** [pengguo318/FJSPDRL#6]
Hi! Impressive work!
In the paper, I found that the input for decoders is the concatenation of three components (see equations 3 and 4). However, it shows to be two instead of three in the code (actor_critic.py, line 178, concateFea = torch.cat((candidate_feature, h_pooled_repeated), dim=-1)). Did I miss the last component?

Another question, why did you use batch size for the number of nodes in the encoder part? (I found this in actor_critic.py, line 101, where the code is self.actor = MLPActor(3, hidden_dim*2, hidden_dim, 1).to(device) ). Here the output of the actor is 1, which means the last layer only has one neuron. Why not use a longer last layer for all the possible actions?

**Comment by Lei-Kun** [comment]
Hello, you can download the latest version of the code at: https://github.com/leikun-starting/End-to-end-DRL-for-FJSP

**Comment by Lei-Kun** [comment]
About the actor and critic network, please refer to Actor1.py.
